### PR TITLE
Updated LLDP Neighbor Detail for matching on some devices that were missing.

### PR DESCRIPTION
**Pull request by jvanderaa**
##### ISSUE TYPE

 - Bugfix Pull Request
 - Additional Testing

##### COMPONENT

cisco_ios_show_lldp_neighbors_detail.template

##### SUMMARY

- After working with @itdependsnetworks we changed the neighbor setup.
- Created raw file with the information to test against what was failing previous

**Before**
```
{"msg": "State Error raised. Rule Line: 47. Input Line: System Name: ZULTYS IP Phone"}
```
**After**
```
        {
            "CAPABILITIES": "T",
            "CHASSIS_ID": "172.16.4.125",
            "LOCAL_INTERFACE": "Gi2/0/13",
            "MANAGEMENT_IP": "",
            "NEIGHBOR": "ZULTYS IP Phone",
            "NEIGHBOR_INTERFACE": "WAN PORT",
            "NEIGHBOR_PORT_ID": "000b.ea00.96e0",
            "SYSTEM_DESCRIPTION": "ZULTYS IP Phone",
            "VLAN": ""
        },
```

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
Can you provide a full output? Not having the headers and footers (total entries) can make it lose context in the future. I realize not all of these have it, but would like to set a better precedence moving forward.

**Comment by jvanderaa** [comment]
I need to delete branch e62f0495f0427b67ea2ee3011cce4e79959ff8f3 from this. Working on figuring out the proper way to do that since I didn't branch properly on the fork.

File: templates/cisco_ios_show_lldp_neighbors_detail.template [hunk 1]
```diff
@@ -2,7 +2,7 @@ Value LOCAL_INTERFACE (\S+)
 Value CHASSIS_ID (\S+)
 Value NEIGHBOR_PORT_ID (.*)
 Value NEIGHBOR_INTERFACE (.*)
-Value NEIGHBOR (\S+)
+Value NEIGHBOR (.+)
```
Comment (jmcgill298) [inline templates/cisco_ios_show_lldp_neighbors_detail.template]:
```suggestion
Value NEIGHBOR (.+?)
```

File: tests/cisco_ios/show_lldp_neighbors_detail/cisco_ios_show_lldp_neighbors_detail4.raw [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    Model: ZIP 33G
+    Capabilities: NP, PD, IN
+    Device type: Endpoint Class III
+    Network Policy(Voice): VLAN 174, tagged, Layer-2 priority: 5, DSCP: 46
+    PD device, Power source: PSE, Power Priority: High, Wattage: 4.0
+    Location - not advertised
+
+
+
+Total entries displayed: 1
```
Comment (jmcgill298) [inline tests/cisco_ios/show_lldp_neighbors_detail/cisco_ios_show_lldp_neighbors_detail4.raw]:
```suggestion
Total entries displayed: 1

```

**Pull request merged**